runtime/ftplugin/nvim-pack.lua

```diff
@@ -49,3 +50,31 @@ for i, l in ipairs(lines) do
     hi_range(i, 4, l:len(), 'DiagnosticHint')
   end
 end
+
+-- Mappings
+local jump = function(direction)
+  local lnum = vim.api.nvim_win_get_cursor(0)[1]
+  local buf_lines = vim.api.nvim_buf_get_lines(0, 0, -1, false)
+  local from, to, by = lnum + 1, #buf_lines, 1
+  if direction == 'prev' then
+    from, to, by = lnum - 1, 1, -1
+  end
+
+  for i = from, to, by do
+    if buf_lines[i]:match('^## ') then
+      vim.api.nvim_win_set_cursor(0, { i, 0 })
```

Comment:
Does `vim.fn.search()` achieve this with less code?

Reply:
Maybe, yeah. If Vimscript functions are generally okay to use in such situations, I'll take a look.

Reply:
Yeah, it does simplify the code quite a bit :+1:  Will change it.